api/src/services/script-runner.service.ts

```diff
@@ -531,40 +587,58 @@ export class ScriptRunnerService {
     });
   }
 
-  async addLotteryTranslationsHelper() {
-    const updateForLanguage = async (
-      language: LanguagesEnum,
-      translationKeys: Record<string, Record<string, string>>,
-    ) => {
-      let translations;
-      translations = await this.prisma.translations.findFirst({
-        where: { language, jurisdictionId: null },
-      });
+  async updateTranslationsForLanguage(
```

Comment:
**question:** This is all looking good but do you mind sharing more on this refactor? Specifically the benefits of findMany and the createIfMissing? Is this just a code improvement or would the previous helper not work for this case?